### PR TITLE
Add anchor CSS

### PR DESCRIPTION
**Pull request by Akarys42**
Add the CSS required for https://github.com/QuiltMC/quiltmc.org/pull/122

### CODE REVIEW

**Review by gdude2002 — CHANGES_REQUESTED** [review]
Small PR, but not sure it matches the style conventions - not sure we have any selectors that long without splitting the line after the brace.

Also, please bump the version.

File: sass/components/anchors.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+:where(h1, h2, h3, h4, h5, h6) a .header-anchor-container .header-anchor { 
+    visibility: hidden;
+}
+:where(h1, h2, h3, h4, h5, h6):hover a .header-anchor-container .header-anchor {
```
Comment (pluiedev) [inline sass/components/anchors.scss]: It's a bit too verbose for my liking — I'd write this instead:
```scss
:where(h1, h2, h3, h4, h5, h6) {
    a .header-anchor-container .header-anchor { 
        visibility: hidden;
    }
    &:hover a .header-anchor-container .header-anchor { 
        visibility: visible;
    }
}

**Review by pluiedev — CHANGES_REQUESTED** [review]

File: sass/components/anchors.scss (hunk 1 above)
Comment (Akarys42) [reply sass/components/anchors.scss]: This is very nitpicky

**Comment by gdude2002** [comment]
Everything's been addressed, so off we go.

**Comment by gdude2002** [comment]
Thanks!

**Pull request merged**